layouts/shortcodes/dbm-mysql-agent-config-examples.md

```diff
@@ -0,0 +1,64 @@
+### One agent connecting to multiple hosts
+To connect to multiple hosts, create an entry for each host in the MySQL integration config.
+```yaml
+init_config:
+instances:
+  - dbm: true
+    host: localhost_A
```

Comment:
```suggestion
    host: products-primary.123456789012.us-east-1.rds.amazonaws.com
```

Reply:
Additionally recommended to include replication for these: https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/data/conf.yaml.example#L204